.eslintrc

```diff
@@ -4,12 +4,14 @@
     "node": true
   },
   "rules": {
-    "no-var": ["off"],
-    "no-param-reassign": ["off"],
-    "vars-on-top": ["off"],
-    "prefer-arrow-callback": ["off"],
-    "prefer-template": ["off"],
-    "comma-dangle": ["error", "never"],
-    "object-curly-spacing": ["error", "never"]
+    "no-var": 0,
```

Comment:
I've come the conclusion that strings are more explicit than number in this case. But of course we don't need arrays.